test/e2e/common/node/lifecycle_hook.go

```diff
@@ -661,3 +662,43 @@ var _ = SIGDescribe(feature.PodLifecycleSleepActionAllowZero, func() {
 
 	})
 })
+
+var _ = SIGDescribe(feature.ContainerStopSignals, func() {
```

Comment:
Since the container runtime implementation for the CRI API change hasn’t been done yet, does that mean this release will only test whether StopSignal can be used at the API level here? It might not be appropriate to bring this up with the exception request deadline approaching, but I was concerned because it doesn’t seem to follow the [Kubernetes feature development and container runtimes policy](https://www.kubernetes.dev/docs/code/cri-api-dev-policies/#same-maturity-level-for-alpha).

Reply:
Thanks for bringing this up! This only tests StopSignal at the API level right now. I can remove the test if it makes more sense to have this once the container runtime also supports StopSignal.